cpp/include/raft/sparse/mst/detail/mst_solver_inl.cuh

```diff
@@ -83,12 +83,12 @@ MST_solver<vertex_t, edge_t, weight_t>::MST_solver(
   //Initially, color holds the vertex id as color
   auto policy = rmm::exec_policy(stream);
   if (initialize_colors_) {
-    thrust::sequence(policy->on(stream), color.begin(), color.end(), 0);
-    thrust::sequence(policy->on(stream), color_index, color_index + v, 0);
+    thrust::sequence(policy, color.begin(), color.end(), 0);
```

Comment:
The errors in CI in this test seem to be coming from a missing 

```C++
#include <rmm/exec_policy.hpp>
```

in this file

Reply:
@divyegala cleaned this up in a different PR for HDBSCAN. I'm guessing the root cause for some of the thrust compilation issues we've been seeing downstream is that MST is some old old API calls. @divye what do you think about cherry-picking that commit from our HDBSCAN branch into a new PR?